ansible/configs/ocp4-cluster/default_vars.yml

```diff
@@ -28,11 +28,12 @@ install_ipa_client: false
 
 # Enable this if you want to create a user on the bastion
 # Mutually exclusive with {{ install_ipa_client }}
-install_student_user: true
+install_student_user: false
 
 # This should be overwritten based on the user ordering the catalog item
 # It will be used by the bastion-student-user role and created on the bastion
 student_name: lab-user
+student_password: Redhat01
```

Comment:
Please revert this change and do not include another config in this PR. If this needs to change, please open an issue or submit a separate PR so the config owner can address.

Also, we shouldn't be including passwords in defaults like this. I'd prefer this to be more like `CHANGEME`

Reply:
ocp4-cluster ain't mine/not using those in my config, these files have been pushed by mistake, sorry for that